docs/manual.md

```diff
@@ -0,0 +1,83 @@
+#Data Exporter
+A set of cli-tools to import and export users from one ownCloud instance
+in to another. The export contains all user-settings, files and shares.
+
+##Use cases
+- Manual zero-downtime migration of users and their shares from one instance in to another.
+- Migrate from instances with different storages (POSIX to S3).
+- Service GDPR-Requests by providing all files and metadata of a user in a single package.
+- Merge users from different instances.
+
+##Usage Example
+We want to export "user1" from a "old" to a "new" instance while preserving all shares with
+users on the old instance. For this example both instance must be able to reach each
+other via federation.
+
+Export the user on the source instance
+
+``$ ./occ export:user user1 /tmp/export``
+
+This will create a folder /tmp/export/user1 which contains
+all the files and metadata of the user.
+
+Copy the export to the target instance for example
+
+``$ scp -rp /tmp/export root@newinstance.com:/tmp/export``
+
+Import the user by running the import command on the new instance
+
+``$ ./occ import:user /tmp/export/user1``
+
+This imports the user in to the new instance while converting all his outgoing-shares
+to federated shares pointing to the old instance.
+
+As user1 now lives on a new instance we need to rewrite all shares so that
```

Comment:
this isn't really a rewrite: new shares will be created pointing to the new instance. The old shares will still be available.